.pre-commit-config.yaml

```diff
@@ -9,7 +9,8 @@ minimum_pre_commit_version: 3.8.0
 files: |
   (?x)^(
       model/.*|
-      tools/.*
+      tools/.*|
+      .github/.*.py
   )$
```

Comment:
Hmm, I haven't realized about this, but why do we have a `files:` filter here? Shouldn't every hook have a specific filter, if needed, and otherwise apply hooks to the whole repo? That's what we do in gt4py at least...

Reply:
I have no idea, why it is like that. Probably for no real reason. I suggest, that we leave it like this here and adress it in a different PR.